.env.sample

```diff
@@ -28,3 +28,4 @@ PUSHER_PORT="8888"
 PUSHER_WS_HOST="127.0.0.1"
 PUSHER_WS_PORT="45449"
 EARLY_V2_EMAIL="@codeminer42.com"
+GOOGLE_ANALYTICS_ID=
```

Comment:
It would be nice to declare this variable here as well as non-required: https://github.com/Codeminer42/cm42-central/blob/master/app.json#L78